.gitignore

```diff
@@ -23,7 +23,6 @@
 hs_err_pid*
 
 # Packages inside files folder
-files/*
 !files/.gitkeep
```

Comment:
Shall we remove this and the .gitkeep file as well?

Reply:
Done https://github.com/wso2/ansible-is/pull/22/commits/0004d29a774d88a385474cf46f5d6c8b525849de